### PR TITLE
feat: add command to test plugins file

### PR DESCRIPTION
**Pull request by scortier**
- Created a command to run plugins test
- Created a GitHub workflow to run plugins test only on main branch

### CODE REVIEW

File: plugins/extractors/elastic/elastic_test.go [hunk 1]
```diff
@@ -214,7 +214,7 @@ func getExpectedVal() []models.Record {
 		}),
 		models.NewRecord(&assetsv1beta1.Table{
 			Resource: &commonv1beta1.Resource{
-				Urn:  "elastic.index2",
```
Comment (StewartJingga) [inline plugins/extractors/elastic/elastic_test.go:217]: is this to fix breaking test?

File: .github/workflows/plugin_test.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Plugin Test
```
Comment (StewartJingga) [inline .github/workflows/plugin_test.yml:1]: Let's rename it to Test Plugins

File: .github/workflows/plugin_test.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (StewartJingga) [inline .github/workflows/plugin_test.yml:13]: Does the default CI image have Docker installer? CMIIW but it is needed for running dockertest.

**Review by StewartJingga — CHANGES_REQUESTED** [review]

File: plugins/extractors/elastic/elastic_test.go (hunk 1 above)
Comment (scortier) [reply plugins/extractors/elastic/elastic_test.go:217]: Yes

File: .github/workflows/plugin_test.yml (hunk 3 above)
Comment (scortier) [reply .github/workflows/plugin_test.yml:13]: Although the test were running fine in CI, let me verify it again

**Pull request merged**